### ISSUE TITLE
Are environment variables possible as an alternative to a stateful config.json file?

### ISSUE THREAD

**Issue by Leopere** [mattermost/focalboard#3069]
https://github.com/mattermost/focalboard/blob/9dbb0c88e1621a0801fe249f87cac20238be8805/docker/docker-compose-db-nginx.yml#L18

**Comment by chenilim** [comment]
@jespino, are automatic env variables implemented in config.go, based on [AutomaticEnv()](https://github.com/mattermost/focalboard/blob/main/server/services/config/config.go#L77)? If so, we may just need to document them better.

**Comment by jespino** [comment]
Exactly, but it only affects the config for personal server and desktop apps, not for plugin mode. In plugin mode all the config comes from Mattermost itself.

**Comment by Leopere** [comment]
Is this something that can be updated?  I really like the idea of using Focalboard.  I ended up getting a copy deployed in our stack successfully but statelessness in a container is really nice.
